src/wasm/cpp/wasm.manifest

```diff
@@ -0,0 +1,42 @@
+schema Data
```

Comment:
This looks like it's just a basic demo, checking since it's named and placed as if it's general-purpose wasm manifest any C++ app would need. Maybe a clearer path/filename?

Reply:
This is a straight copy of the cpp demo already checked in